src/Http/Controllers/VerificationController.php

```diff
@@ -82,14 +80,8 @@ public function sendNewCode(Request $request)
      */
     public function verify(Request $request)
     {
-        if (Verification::verifyCode(Auth::user(), $request->get('action_name'), $request->get('code'))) {
-            if (Config::get('verifications.actions.'. $request->get('action_name') .'.keep_verified_during_session')) {
-                Session::put('last_activity', Carbon::now()->toDateTime());
-            }
-
-            return Redirect::to($request->get('redirectToUrl'))->with('success', trans('brackets/verifications::verifications.code_verify_success'));
-        }
-
-        return Redirect::back()->with('error', trans('brackets/verifications::verifications.code_verify_failed'));
+        return Verification::verifyCode(Auth::user(), $request->get('action_name'), $request->get('code'))
+            ? Redirect::to($request->get('redirectToUrl'))->with('success', trans('brackets/verifications::verifications.code_verify_success'))
```

Comment:
Toto mi pride prilis dlhy zapis, aby to bolo cez ternary operator, je to tazko citatelne

Cize teraz sa kde updatuje ta last_activity?